src/org/jitsi/impl/neomedia/codec/video/h264/JNIEncoder.java

```diff
@@ -614,7 +614,6 @@ else if ("h264.profile".equals(k))
             FFmpeg.avcodeccontext_add_flags2(avctx,
                 FFmpeg.CODEC_FLAG2_INTRA_REFRESH);
         }
-        FFmpeg.avcodeccontext_set_me_method(avctx, 7);
```

Comment:
This will need and changing the natives at some point, removing that method...

Reply:
Well, one could keep them just for older FFmpeg. For FFmpeg 4, I created conditionals in that other Pull Request.